### PR TITLE
Generate and display CV thumbnails

### PR DESCRIPTION
**Pull request by andylolz**
Fixes (ish) #33.

Currently lacking tests…

### CODE REVIEW

**Comment by frabcus** [comment]
I like it, and I need to go to sleep!

Two obvious things I'd do:

1) Make the thumbs a big bigger vertically. I think they should have same aspect ratio as an A4 sheet so they look like first page of a CV at a glance. (I realise the current ones don't do that!)

2) Make the actual images into links. It's possible we can get rid of the "view full page" link. In fact, I really hope we can! If they are links with a border (either all the time and/or when hover, not sure which!) people will have to click them anyway to see more.

File: templates/all_cvs.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
             {% endfor %}
         </div>
 
-        <nav>
+        <nav class="text-center">
```
Comment (frabcus) [inline templates/all_cvs.html:19]: This looks so much better centered!

**Comment by frabcus** [comment]
Also, if I was to deploy to live what cron stuff do I need?

I think it is fine if front page / all page don't show CVs until they get their thumb (those pages both have a cache in Flask anyway). 

I really want to use the thumbs on the `/candidates` page too, so thumbs updating as soon as new uploads appear would be valuable. Am guessing they are a bit slow though?

**Comment by frabcus** [comment]
Also, on my machine the front page images look fuzzy because they are big. Question is if we need big thumbs or not? We could just change front page to show small thumbs and be done. Add big ones later if needed.

Or generate big ones anyway now.

**Comment by andylolz** [comment]
~~The thumbnail generation **isn’t working on heroku** at the moment! I’m not sure why… I’ll keep working on this – it’s not ready for merge yet.~~ UPDATE: Works now (using phantomjs 2).

~~I’ve sorted some of these:~~ UPDATE: All done!
- [x] same aspect ratio as an A4 sheet
- [x] Add cron info to README
- [x] Make the actual images into links
- [x] front page / all page don't show CVs until they get their thumb
- [x] ~~generate bigger thumbs (for frontpage)~~ make everything a bit smaller instead

> Am guessing they are a bit slow though?

Yeah, they take ages. I told scheduler to run every 10 mins. CVs won’t be shown on front page / all CVs page until the thumb is present. On constituency page, if a thumb hasn’t yet been generated you’ll see google docs viewer.

**Comment by frabcus** [comment]
It all looks good to me. Will need rebasing with https://github.com/frabcus/mpcv/pull/40/files

**Comment by andylolz** [comment]
Ace – thanks!

**Pull request closed without merging**